v2.0/create-table.md

```diff
@@ -205,45 +205,82 @@ We also have other resources on indexes:
 
 {% include faq/auto-generate-unique-ids_v1.1.html %}
 
-### Create a Table with Foreign Keys
+### Create a Table with a Foreign Key Constraint
 
-[Foreign keys](foreign-key.html) guarantee a column uses only values that already exist in the column it references, which must be from another table. This constraint enforces referential integrity between the two tables.
+[Foreign key constraints](foreign-key.html) guarantee a column uses only values that already exist in the column it references, which must be from another table. This constraint enforces referential integrity between the two tables.
 
 There are a [number of rules](foreign-key.html#rules-for-creating-foreign-keys) that govern foreign keys, but the two most important are:
 
 - Foreign key columns must be [indexed](indexes.html) when creating the table using `INDEX`, `PRIMARY KEY`, or `UNIQUE`.
 
 - Referenced columns must contain only unique values. This means the `REFERENCES` clause must use exactly the same columns as a [Primary Key](primary-key.html) or [Unique](unique.html) constraint.
 
-In this example, we'll show a series of tables using different formats of foreign keys.
+<span class="version-tag">New in v2.0:</span> You can include a [foreign key action](foreign-key.html#foreign-key-actions) to specify what happens when a column referenced by a foreign key is updated or deleted. The default actions are `ON UPDATE NO ACTION` and `ON DELETE NO ACTION`.
 
-~~~ sql
-> CREATE TABLE customers (id INT PRIMARY KEY, email STRING UNIQUE);
+In this example, we use `ON DELETE CASCADE` (i.e., when row referenced by a foreign key constraint is deleted, all dependent rows are also deleted).
 
-> CREATE TABLE products (sku STRING PRIMARY KEY, price DECIMAL(9,2));
+{{site.data.alerts.callout_danger}}<code>CASCADE</code> does not list objects it drops, so it should be used cautiously.{{site.data.alerts.end}}
```

Comment:
I'd remove this. When mentioning that this example will use `CASCADE`, you already explain what that means.

Reply:
Done